proposals/lip-0066.md

```diff
@@ -119,7 +127,7 @@ def childKeyDerivation(node: ExtendedKey, index: uint32) -> ExtendedKey:
 
 The path for deriving the n-th private key from that phrase is `m/44'/134'/n'`. This follows the specifications of [BIP 44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki#Registered_coin_types) and [SLIP 44](https://github.com/satoshilabs/slips/blob/master/slip-0044.md), assigning 134 to the LSK token.
 
-The path for deriving the private key of the n-th [generator key](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#generator-key) pair, for the chain with [ID](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0043.md#chain-id) `chainID `is `m/25519'/134'/chainID'/n'.` If `chainID` is not known, use the path `m/25519'/134'/0'/n'` for the n-th generator key derived for chains without known chain ID.
+The path for deriving the private key of the n-th [generator key](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#generator-key) pair, for the chain with [ID](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0043.md#chain-id) `ChainID` is `m/25519'/134'/ChainIDConverted'/n'`, where `ChainIDConverted` is the return integer value of `uint32be(chainID)`. The hardened derivation `chainIDConverted'` can always be converted to a byte value that fits into 4 bytes because the first bit is set to `0`.
```

Comment:
```suggestion
The path for deriving the private key of the n-th [generator key](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md#generator-key) pair, for the chain with [ID](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0043.md#chain-id) `chainID` is `m/25519'/134'/chainIDConverted'/n'`, where `chainIDConverted` is the return integer value of `uint32be(chainID)`. The hardened derivation `chainIDConverted'` can always be converted to a byte value that fits into 4 bytes because the first bit is set to `0`.
```

Reply:
Updated the text.